libs/community/langchain_community/chains/graph_qa/cypher.py

```diff
@@ -311,12 +311,15 @@ def _call(
         _run_manager = run_manager or CallbackManagerForChainRun.get_noop_manager()
         callbacks = _run_manager.get_child()
         question = inputs[self.input_key]
+        args = {
+            "question": question,
+            "schema": self.graph_schema,
+        }
+        args.update(inputs)
```

Comment:
That's a good question, regarding the content duplication, if the input_key for the question is different than "question" it will indeed duplicate because the default template uses "question" and I cant guarantee that the user didn't specified other `cypher_prompt` template.